oauth2/authorize.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+/**
+ * Authorization Server Member
+ *
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ * @author    Jerry Padgett <sjpadgett@gmail.com>
+ * @copyright Copyright (c) 2020 Jerry Padgett <sjpadgett@gmail.com>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+require_once(__DIR__ . "/../_rest_config.php");
+
+$gbl = RestConfig::GetInstance();
+if (empty($gbl::$SITE)) {
+    http_response_code(401);
+    exit;
+}
+// strange and frustrating that globals session isn't maintained from cross origin
+// to our endpoint thus, below.
```

Comment:
Is there a connection going to another site somewhere? (If so, then likely is the session SameSite setting)